cdap-hbase-compat-1.1/src/main/java/co/cask/cdap/data2/replication/hbase11/LastReplicateTimeObserver.java

```diff
@@ -65,10 +64,8 @@ public void postReplicateLogEntries(ObserverContext<RegionServerCoprocessorEnvir
                 entry.getKey().getTableName().toStringUtf8(),
                 entry.getKey().getWriteTime(),
                 entry.getKey().getEncodedRegionName().toStringUtf8());
-      if (hBase11TableUpdater != null) {
-        hBase11TableUpdater.updateTime(entry.getKey().getEncodedRegionName().toStringUtf8(),
-                                       entry.getKey().getWriteTime());
-      }
+      hBase11TableUpdater.updateTime(entry.getKey().getEncodedRegionName().toStringUtf8(),
```

Comment:
did you see the earlier comments about this? I think I've commented on this 3 times. In general, you can't just assume a byte[] is a utf8 string. From a brief look at the code, it looks like you should be using HRegionInfo.encodeRegionName().